src/components/HomePage/HomePage.js

```diff
@@ -42,16 +42,16 @@ const HomePage = () => {
+    const bigBookArray = [...bookList, ...bookList, ...bookList, ...bookList, ...bookList, ...bookList, ...bookList, ...bookList, ...bookList, ...bookList, ...bookList, ...bookList,...bookList, ...bookList]
+
     return (
         <div>
             <h1>This is the Home Page</h1>
 
             {/* Render out the booklList on the home page */}
+            <BookList bookList={booklist} />
```

Comment:
When I run application I have an error: `ERROR in [eslint] 
src/components/HomePage/HomePage.js
  Line 52:33:  'booklist' is not defined  no-undef
Search for the keywords to learn more about each error.`
Could you please delete line 52?

Reply:
Could you please fix line 157. I suppose it should be `{bookList} `instead of `{booklist}`.